tools/firmware-utils/src/ptgen.c

```diff
@@ -52,6 +52,8 @@
 #define swap(a, b) \
 	do { typeof(a) __tmp = (a); (a) = (b); (b) = __tmp; } while (0)
 
+#define BIT(_x)		(1UL << (_x))
```

Comment:
firmware-utils has been moved into a separate repository recently.

Please discuss with @ynezz and/or @rmilecki how submission should be handled there.